package.json

```diff
@@ -6,6 +6,8 @@
     "lint": "lerna run lint",
     "test": "lerna run test",
     "install": "npm run bootstrap",
-    "bootstrap": "lerna bootstrap"
+    "bootstrap": "lerna bootstrap",
+    "build": "lerna run build",
+    "build:watch": "lerna run --parallel build -- -- --watch"
```

Comment:
Note: the first `--` is to separate lerna args from the npm command args, and the second is to separate the npm args from the script args.